src/main/java/nl/hr/recipefinder/service/RecipeService.java

```diff
@@ -0,0 +1,7 @@
+package nl.hr.recipefinder.service;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class RecipeService {
+}
```

Comment:
You should probably already link the `RecipeRepository` 

```suggestion
public class RecipeService {
    private RecipeRepository recipeRepository;
    
    public RecipeService(RecipeRepository recipeRepository) {
        this.recipeRepository = recipeRepository;
    }
}
```